test/kpod_images.bats

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bats
+
+load helpers
+
+IMAGE="alpine:latest"
```

Comment:
This doesn't appear to be used in this file - it could be set to `debian:6.0.10` and used instead where that image's name is hard-coded, or dropped.